src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
What's this file for?

Reply:
I think it is used to expose useful types from react-scripts to make it easier to work with images for example.

Reply:
Yes, it's automatic and we should leave it, for ts.